db/schema.rb

```diff
@@ -1059,7 +1072,7 @@
     t.datetime "updated_at", precision: nil, null: false
     t.bigint "parent_transaction_id"
     t.index ["parent_transaction_id"], name: "fk_rails_6ad225b020"
-    t.index ["status"], name: "index_stripe_transactions_on_status"
+    t.index ["status"], name: "index_stripe_records_on_status"
```

Comment:
Strictly speaking, you're not renaming this index as part of the migration. It would seem that you manually edited this into `schema.rb`, which is problematic because then the existing index in prod won't actually be renamed.

There is no `rename_index` method to my knowledge (Google may correct me on this one, though) and you probably have to run two separate `drop_index` (or was it `delete_index`? I can never remember sorry...) and `create_index` statements.

Reply:
TIL: The indexes are actually automatically migrated! :star_struck: